src/libImaging/Unpack.c

```diff
@@ -1560,299 +1559,294 @@ static struct {
 
     /* floating point variations */
-    {"F", "F", 32, copy4},
-    {"F", "F;8", 8, unpackF8},
-    {"F", "F;8S", 8, unpackF8S},
-    {"F", "F;16", 16, unpackF16},
-    {"F", "F;16S", 16, unpackF16S},
-    {"F", "F;16B", 16, unpackF16B},
-    {"F", "F;16BS", 16, unpackF16BS},
-    {"F", "F;16N", 16, unpackF16N},
-    {"F", "F;16NS", 16, unpackF16NS},
-    {"F", "F;32", 32, unpackF32},
-    {"F", "F;32S", 32, unpackF32S},
-    {"F", "F;32B", 32, unpackF32B},
-    {"F", "F;32BS", 32, unpackF32BS},
-    {"F", "F;32N", 32, unpackF32N},
-    {"F", "F;32NS", 32, unpackF32NS},
-    {"F", "F;32F", 32, unpackF32F},
-    {"F", "F;32BF", 32, unpackF32BF},
-    {"F", "F;32NF", 32, unpackF32NF},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F, 32, copy4},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_8, 8, unpackF8},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_8S, 8, unpackF8S},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16, 16, unpackF16},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16S, 16, unpackF16S},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16B, 16, unpackF16B},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16BS, 16, unpackF16BS},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16N, 16, unpackF16N},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_16NS, 16, unpackF16NS},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32, 32, unpackF32},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32S, 32, unpackF32S},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32B, 32, unpackF32B},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32BS, 32, unpackF32BS},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32N, 32, unpackF32N},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32NS, 32, unpackF32NS},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32F, 32, unpackF32F},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32BF, 32, unpackF32BF},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_32NF, 32, unpackF32NF},
 #ifdef FLOAT64
-    {"F", "F;64F", 64, unpackF64F},
-    {"F", "F;64BF", 64, unpackF64BF},
-    {"F", "F;64NF", 64, unpackF64NF},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_64F, 64, unpackF64F},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_64BF, 64, unpackF64BF},
+    {IMAGING_MODE_F, IMAGING_RAWMODE_F_64NF, 64, unpackF64NF},
 #endif
 
     /* storage modes */
-    {"I;16", "I;16", 16, copy2},
-    {"I;16B", "I;16B", 16, copy2},
-    {"I;16L", "I;16L", 16, copy2},
-    {"I;16N", "I;16N", 16, copy2},
-
-    {"I;16", "I;16B", 16, unpackI16B_I16},
-    {"I;16", "I;16N", 16, unpackI16N_I16},   // LibTiff native->image endian.
-    {"I;16L", "I;16N", 16, unpackI16N_I16},  // LibTiff native->image endian.
-    {"I;16B", "I;16N", 16, unpackI16N_I16B},
+    {IMAGING_MODE_I_16, IMAGING_RAWMODE_I_16, 16, copy2},
+    {IMAGING_MODE_I_16B, IMAGING_RAWMODE_I_16B, 16, copy2},
+    {IMAGING_MODE_I_16L, IMAGING_RAWMODE_I_16L, 16, copy2},
+    {IMAGING_MODE_I_16N, IMAGING_RAWMODE_I_16N, 16, copy2},
 
-    {"I;16", "I;16R", 16, unpackI16R_I16},
+    {IMAGING_MODE_I_16, IMAGING_RAWMODE_I_16B, 16, unpackI16B_I16},
+    {IMAGING_MODE_I_16, IMAGING_RAWMODE_I_16R, 16, unpackI16R_I16},
```

Comment:
```suggestion
    {IMAGING_MODE_I_16B, IMAGING_RAWMODE_I_16N, 16, unpackI16N_I16B},
    {IMAGING_MODE_I_16, IMAGING_RAWMODE_I_16R, 16, unpackI16R_I16},
```
Restored accidentally removed line.